### PR TITLE
REDESIGN [7701] replace project header with BO

### PR DESCRIPTION
**Pull request by philli-m**
Icon still present in follow button, need a4 refactor to be fixed but that can be done in follow story

### CODE REVIEW

File: meinberlin/templates/base.html
Comment (philli-m) [inline meinberlin/templates/base.html:33]: This is my first attempt to separate out the styles as the dashboard is getting affected, I don't think this is a perfect solution but to keep everything workable and to notice when deleted or altered styles will be problematic in the dashboard I think its a good idea but also happy for other suggestions?

File: meinberlin/templates/base.html
Comment (philli-m) [inline meinberlin/templates/base.html:60]: it's not confirmed if we need their header and footer in the dashboard yet, if so that makes the solution above non viable but also I don't know why we would need a platform header in a CMS

File: meinberlin/templates/base.html
Comment (m4ra) [reply meinberlin/templates/base.html:33]: A more standard way is to work with blocks. E.g here you can make use of the above extra_css block and point to the berlin CSS, and in the meinberlin/templates/a4dashboard/base_dashboard.html you override it with the a4 CSS.

File: meinberlin/templates/base.html
Comment (m4ra) [reply meinberlin/templates/base.html:60]: similar here, make use of the blocks for overriding between templates

File: meinberlin/templates/base.html
Comment (m4ra) [inline meinberlin/templates/base.html:91]: same

**Review by m4ra — CHANGES_REQUESTED** [review]
I went through the template blocks mostly. The styling review cannot really do it as am not part of the design stories process.

**Comment by philli-m** [comment]
@m4ra thank you so much! i'll change those out to use blocks :) 
For style review, I think @goapunk has just been looking at the design from the story and comparing but also Marcelo will also check it once it's on dev anyway

**Review by m4ra — APPROVED** [review]

**Comment by m4ra** [comment]
@philli-m LGTM, I don't know if you wanna wait for julian's review too. If not go ahead and merge.

**Pull request merged**
